src/blk/kernel/KernelDevice.h

```diff
@@ -52,7 +52,6 @@ class KernelDevice : public BlockDevice,
   aio_callback_t discard_callback;
   void *discard_callback_priv;
   bool aio_stop;
-  bool discard_stop;
```

Comment:
I like the idea of eliminating this member class. And I'd like to do that even if we keep using _discard_update_threads() from _discard_stop. One can just make it a parameter for _discard_update_threads() instead.

Reply:
Get